src/chrome-launcher.ts

```diff
@@ -90,6 +85,19 @@ async function launch(opts: Options = {}): Promise<LaunchedChrome> {
   return {pid: instance.pid!, port: instance.port!, kill, process: instance.chrome!};
 }
 
+async function killAll(): Promise<boolean> {
+  let noErrors = true;
+  for (const instance of instances) {
+    try {
+      await instance.kill();
+      instances.delete(instance);
```

Comment:
is it intentional to only remove the instance from the set after it's been successfully killed?

on the one hand, it'd be nice for a potential future `killAll` to try again

on the other hand, if it's erroring because it's already dead it's annoying to have this permanently erroring unnecessarily

I think we just need some separate better upkeep of our `instances` set as processes get killed/die on their own buuuuuuuut

either way let's add a comment here about the removal behavior, so future readers know what's up :)

Reply:
it was intentional in order to retry. but good point about the error being caused by already dead instance, I didn't think of that. 

I'll add a comment